src/main/webapp/script.js

```diff
@@ -22,4 +36,28 @@ function createIFrame(text) {
   const liElement = document.createElement('iframe');
   liElement.src = text;
   return liElement;
-}
\ No newline at end of file
+}
+
+function displayCalendar() {
+    var src="https://calendar.google.com/calendar/embed?height=600&amp;wkst=1&amp;bgcolor=%23ffffff&amp;ctz=America%2FLos_Angeles&amp;src=bWFyaWNhcm9sQGdvb2dsZS5jb20&amp;src=Y19pZmpiYzdmZ21ha25zZ3ZpMmlmc2xqcXBoOEBncm91cC5jYWxlbmRhci5nb29nbGUuY29t&amp;color=%23039BE5&amp;color=%237986CB";
+
+    const calendarElement = document.getElementById('calendar-display');
+    calendarElement.innerHTML = '';
+    calendarElement.appendChild(createCalendarIFrame(src));
+
+}
+
+function createCalendarIFrame(src) {
```

Comment:
Maybe instead of manually doing this, we could have an empty src iframe tag in out HTML. So maybe in our HTML we could have:

 <iframe id="servlet-calender" **_src=""_** style="border:solid 1px #777" width="800" height="600" frameborder="0" scrolling="no"></iframe>

document.getElementById("servlet-calendar").src = decoded;

Then we can fill in the source property. But otherwise looks good!